### PR TITLE
chagne pdf property dialog to modal

### PR DESCRIPTION
**Pull request by ShikeChen-MS**
Currently, PDF property display as an dialog which in the background, viewing function is still available (scrolling, change pages etc.)  Changes in this pull request changes it to modal and the style stays the same, as a result in the background, viewing function is blocked.

### CODE REVIEW

File: web/viewer.css [hunk 1]
```diff
@@ -1541,18 +1541,43 @@ html[dir="rtl"] .treeItemToggler::before {
   box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
 }
 
+#overlayContainer > .container > .modal {
```
Comment (seahawksq) [inline web/viewer.css:1544]: Could you please test the css settings in RTL language and see if display looks well?

**Review by seahawksq — APPROVED** [review]
LGTM, please make sure you convert RTL on modal dialog.

**Comment by ShikeChen-MS** [comment]
Just verified in Arabic(U.A.E) language. Didn't see any visual difference. Seems Github is not working properly this morning (I can not login at all). Sending a email.

Thanks
Shike Chen

**Comment by ShikeChen-MS** [comment]
@seahawksq, I do not have write access to this repository, so can you merge this branch please!

**Review by seahawksq — APPROVED** [review]
LGTM

**Pull request merged**